swatch-core/build.gradle

```diff
@@ -4,6 +4,10 @@ plugins {
 }
 
 dependencies {
+
+    implementation "com.splunk.logging:splunk-library-javalogging:1.11.0"
+    implementation "org.codehaus.janino:janino:3.1.6"
```

Comment:
The janino library is used for conditional processing in the logback xml file.

Reference: http://logback.qos.ch/manual/configuration.html

Reply:
These should be defined in `build.gradle`'s `dependencyManagement` section.  Then we can reference them here like

```
implementation "com.splunk.logging:splunk-library-javalogging"
implementation "org.codehaus.janino:janino"
```

This is to make sure that the entire project uses exactly the same versions